components/lib/utils/DomHandler.js

```diff
@@ -647,10 +646,8 @@ export default class DomHandler {
              * @param {HTMLElement} node - Element to add
              */
             const addScrollableParent = (node) => {
-                if (hideOverlaysOnDocumentScrolling) {
-                    // For document/body/html elements, add window instead
-                    scrollableParents.push(node.nodeName === 'BODY' || node.nodeName === 'HTML' || node.nodeType === 9 ? window : node);
-                }
+                // For document/body/html elements, add window instead
+                scrollableParents.push(node.nodeName === 'BODY' || node.nodeName === 'HTML' || node.nodeType === 9 ? window : node);
```

Comment:
For readability i think we should add a `DomHandler.isDocumentNode(target)` method that does this `node.nodeType === 9` for readability. Then everywhere you use `node.nodeType === 9` change to use this new method which will make the code much more readable for eveyrone!

Reply:
Makes sense... commit is coming